asn1acn.cpp

```diff
@@ -109,6 +119,29 @@ ExtensionSystem::IPlugin::ShutdownFlag Asn1AcnPlugin::aboutToShutdown()
     return SynchronousShutdown;
 }
 
+void Asn1AcnPlugin::loadSettings()
```

Comment:
Since real implementation will most likely consist of new class which will need to be instantiated in plugins initialize() function,  placing fake settings loading here and not in serviceprovider is a good idea imo, as working on real implementation will not affect serviceprovider, but only asn1acn.cpp file than.